src/components/Validate/index.tsx

```diff
@@ -256,15 +276,15 @@ const Validate = ({ toggleSnackbar, setSnackbarMessage }: IHomeProps) => {
               Do you know anything about this?
             </Typography>
             <Box sx={{ width: '95%' }}>
-              <Tooltip title='What aspect is this rating about?' placement='right' arrow>
+              <Tooltip title='How do you know about it?' placement='right' arrow>
```

Comment:
this is how known not aspect, please check @AmrNabel

Reply:
yeh you are right